### PR TITLE
Mocked data improvements

### PR DESCRIPTION
**Pull request by tinybike**

### CODE REVIEW

**Comment by tinybike** [comment]
Note: this PR works with https://github.com/AugurProject/augur-contracts/commit/798472ad6daeabc919cc35d41dfed1b29895c514 -- holding off on updating the augur-contracts version in package.json for now b/c it seems like the versioning is a bit of a mess!

**Comment by epheph** [comment]
Great approach, should save a lot of hearth-ache for future changes and something we should push more widely. Any reason not to tackle the indexed fields too?

**Comment by tinybike** [comment]
Not really -- just did the non-indexed ones because converting between the serialized log data and the human-readable parameters is sucky.  The indexed ones are easier to read, they're just int256s.

**Comment by epheph** [comment]
Sure, there's also fewer of them and they are less likely to change.

**Comment by tinybike** [comment]
Update: augur.js tests are now congruent with https://github.com/AugurProject/augur-contracts/blob/augur-core/feature/deploy_aura

File: src/format/log/format-log-message.js [hunk 1]
```diff
@@ -16,11 +17,17 @@ function formatLogMessage(contractName, eventName, message) {
...
             extraInfo = null;
           }
-          return assign({}, message, {
+          var formattedMessage = {
             extraInfo: extraInfo,
             marketCreationFee: speedomatic.unfix(message.marketCreationFee, "string"),
             topic: decodeTag(message.topic),
-          });
+          };
+          if (message.marketType === 1) {
```
Comment (epheph) [inline src/format/log/format-log-message.js]: surprised we don't have a constant for this, perhaps i'll add it in a separate PR.

Comment (tinybike) [reply src/format/log/format-log-message.js]: Agree, that would be good!

**Comment by tinybike** [comment]
Going ahead and merging this even though travis isn't passing -- failure is just b/c the tests need to be run vs the deploy_aura augur-contracts (verified this locally).

**Pull request merged**